src/content/getusermedia/resolution/js/main.js

```diff
@@ -49,6 +50,10 @@ fourKButton.onclick = function() {
   getMedia(fourKConstraints);
 };
 
+eightKButton.onclick = function() {
```

Comment:
Stuff like this could use fat arrow syntax:

`eightKButton.onclick = () => {...}`